### PR TITLE
[MS] Update About View

### PR DESCRIPTION
**Pull request by fabienSvtr**
- Update links in About View
- Update UI according to mockups (About Page & Version in Homepage)

#5592

### CODE REVIEW

**Review by Max-7 — APPROVED** [review]

**Review by NicoTuxx — CHANGES_REQUESTED** [review]
Multiple design problems on About page:
- [ ] The 'update' button colors for hover and focus/clicked states create weird effects (look in the linked video)
- [ ] The links have button appearance (Parsec Cloud, License...) why not, but the 'show changelog' who's behavior is a button one, has a link appearance, this is confusing for a user.
- [ ] The home page modal version has 2 titles, the 'about' and 'Parsec Cloud' just one after each other looks weird. (cf linked image)


https://github.com/Scille/parsec-cloud/assets/19772607/ea445a8d-a8b0-4d20-afe0-8656f4447a1d

image

File: client/src/locales/en-US.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
         "projectLink": "https://parsec.cloud",
         "projectSources": "https://github.com/Scille/parsec-cloud",
         "license": "BUSL-1.1",
-        "licenseLink": "https://spdx.org/licenses/BUSL-1.1.html"
+        "licenseLink": "https://github.com/Scille/parsec-cloud/blob/master/LICENSE"
```
[the remark this answers is not in the document]
Comment (NicoTuxx) [reply client/src/locales/en-US.json:9]: It's okay for the moment, I opened a new issue to take care of this specific point.

**Review by NicoTuxx — APPROVED** [review]

File: client/src/locales/en-US.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
         "projectLink": "https://parsec.cloud",
         "projectSources": "https://github.com/Scille/parsec-cloud",
         "license": "BUSL-1.1",
-        "licenseLink": "https://spdx.org/licenses/BUSL-1.1.html",
+        "licenseLink": "https://github.com/Scille/parsec-cloud/blob/master/LICENSE",
```
Comment (FirelightFlagboy) [inline client/src/locales/en-US.json:9]: Use raw link ?

```suggestion
        "licenseLink": "https://raw.githubusercontent.com/Scille/parsec-cloud/master/LICENSE",
```

File: client/src/locales/fr-FR.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
         "projectLink": "https://parsec.cloud",
         "projectSources": "https://github.com/Scille/parsec-cloud",
         "license": "BUSL-1.1",
-        "licenseLink": "https://spdx.org/licenses/BUSL-1.1.html",
+        "licenseLink": "https://github.com/Scille/parsec-cloud/blob/master/LICENSE",
```
Comment (FirelightFlagboy) [inline client/src/locales/fr-FR.json:9]: Use raw link ?

```suggestion
        "licenseLink": "https://raw.githubusercontent.com/Scille/parsec-cloud/master/LICENSE",
```

**Pull request merged**